src/OpenTelemetry.AutoInstrumentation.Native/stack_capture_strategy_factory.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "stack_capture_strategy_factory.h"
+#include "dot_net_stack_capture_strategy.h"
+#ifdef _WIN64
+#include "netfx_stack_capture_strategy.h"
+#endif
+
+namespace continuous_profiler
+{
+
+std::unique_ptr<IStackCaptureStrategy> StackCaptureStrategyFactory::Create(ICorProfilerInfo2* profilerInfo,
+                                                                           const trace::RuntimeInformation& runtimeInfo)
+{
+
+    if (runtimeInfo.is_desktop())
+    {
+#ifdef _WIN64
```

Comment:
Our code is specifically X64, so better to check `_M_AMD64` - it will help if we ever create windows ARM64 build, which will use the same path otherwise.